src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -243,6 +243,9 @@ if(CLR_CMAKE_TARGET_WIN32)
         add_custom_target(inject_debug_resources_singlefilehost ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/inject_debug_resources_singlefilehost.timestamp)
         add_dependencies(runtime inject_debug_resources_singlefilehost)
     endif()
+else()
+    add_custom_target(inject_debug_resources_coreclr DEPENDS coreclr)
```

Comment:
The other thing that we could do is make runtimeinfo depend on inject_debug_resources_coreclr on windows, given that conceptually there's no external resources needed for debugging coreclr. Do you have a preference between the dependencies vs this phony target?

Reply:
I'd rather do the phony target so we don't need to add a different dependency that has the same effect on non-Windows platforms.